src/main/java/com/cloudbees/jenkins/GitHubCommitNotifier.java

```diff
@@ -44,42 +37,50 @@
 
     private final String resultOnFailure;
     private static final Result[] SUPPORTED_RESULTS = {FAILURE, UNSTABLE, SUCCESS};
-    
+
     @DataBoundConstructor
     public GitHubCommitNotifier(String resultOnFailure) {
         this.resultOnFailure = resultOnFailure;
     }
-    
+
     @Deprecated
     public GitHubCommitNotifier() {
         this(getDefaultResultOnFailure().toString());
     }
 
-    public @Nonnull String getResultOnFailure() {
+    @Nonnull
+    public String getResultOnFailure() {
         return resultOnFailure != null ? resultOnFailure : getDefaultResultOnFailure().toString();
     }
-     
-    public static @Nonnull Result getDefaultResultOnFailure() {
+
+    @Nonnull
+    public static Result getDefaultResultOnFailure() {
         return SUPPORTED_RESULTS[0];
     }
-    
-    /*package*/ @Nonnull Result getEffectiveResultOnFailure() {
+
+
+    @Nonnull
+    /*package*/ Result getEffectiveResultOnFailure() {
         if (resultOnFailure == null) {
             return getDefaultResultOnFailure();
         }
-        
+
         for (Result result : SUPPORTED_RESULTS) {
-            if (result.toString().equals(resultOnFailure)) return result;
+            if (result.toString().equals(resultOnFailure)) {
+                return result;
+            }
         }
         return getDefaultResultOnFailure();
     }
-    
+
     public BuildStepMonitor getRequiredMonitorService() {
         return BuildStepMonitor.NONE;
     }
 
     @Override
-    public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) throws InterruptedException, IOException {
+    public boolean perform(AbstractBuild<?, ?> build,
```

Comment:
line breaks in method sign not mentioned in checkstyle - in one place its good to move arguments on new lines, in other - break line by `throws`. So this can be done in any way